rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java

```diff
@@ -42,6 +42,7 @@
     public static final String INT = "int";
     public static final String INT256 = "int256";
     public static final String UINT = "uint";
+    public static final String UINT256 = "uint256";
```

Comment:
Are you sure that we don't need any modification in static class IntType?

Reply:
I Have Checked Thoroughly... No Other Modification Needed

If The getCanonicalName() Gets Called.. The Parent SolidityType Returns The name, Which I Think Is Fine